Felienne/hedy
# [BUG FIX] Fix not showing of save/share option on corresponding tutorial step

**Description**
The save/share option on the topbar was hidden due to the tutorial still being on the quiz tab. We switch back to the default tab in this step to make sure the banner is shown as expected.

**Fixes _issue or discussion number_**
This PR fixes #3225.

**How to test**
Walkthrough the tutorial and verify that the save/share option is correctly shown on the corresponding step.
